dom.bs

```diff
@@ -2839,6 +2839,7 @@ interface mixin ParentNode {
 
   [CEReactions, Unscopable] void prepend((Node or DOMString)... nodes);
   [CEReactions, Unscopable] void append((Node or DOMString)... nodes);
+  [CEReactions, Unscopable] void replaceContents((Node or DOMString)... nodes);
```

Comment:
I think per the discussion we ended up with `replaceChildren()` as a name, as "contents" doesn't always refer to nodes. Did I miss something?